composer.json

```diff
@@ -4,34 +4,29 @@
   "homepage": "https://wordpress.org/plugins/stream/",
   "type": "wordpress-plugin",
   "license": "GPL-2.0-or-later",
-  "repositories": [
-    {
-      "type": "git",
-      "url": "https://github.com/WordPress/wordpress-develop.git"
-    }
-  ],
   "require": {
     "composer/installers": "~1.0"
   },
   "require-dev": {
+    "johnpbloch/wordpress": "^5.4",
     "php-coveralls/php-coveralls": "^2.1",
     "phpunit/phpunit": "^5.7",
-    "wordpress/wordpress": "^5.2",
     "wp-cli/wp-cli-bundle": "^2.2",
     "wp-coding-standards/wpcs": "^2.2",
-    "wpsh/local": "^0.2.3",
-    "xwp/wp-dev-lib": "^1.5"
+    "wp-phpunit/wp-phpunit": "^5.4",
+    "wpsh/local": "^0.2.3"
   },
   "config": {
+    "process-timeout": 600,
     "sort-packages": true,
     "platform": {
-      "php": "5.6"
+      "php": "5.6.20"
     }
   },
+  "extra": {
+    "wordpress-install-dir": "local/public"
```

Comment:
Previously we were using the WP core files provisioned by the `wordpress` docker container during the `entrypoint` setup. We now have full control of the source files used for development which is much more reliable and provisions the container much faster, too.